pkg/util/preconditions.go

```diff
@@ -19,18 +19,21 @@ func ValidateEnvironment(ctx context.Context) error {
 	}
 
 	var errs *multierror.Error
-	_, err = exec.LookPath("docker")
+	_, err = exec.LookPath("docker-compose")
```

Comment:
we're using `docker compose` rather than `docker-compose` -- it is the direction the project is moving seems like

Reply:
ooo duh ok. I will remove the docker-compose part.

Reply:
This is removed, but I have kept the function from panicing.